legend-depot-store-mongo/src/main/java/org/finos/legend/depot/store/mongo/admin/metrics/QueryMetricsMongo.java

```diff
@@ -71,6 +71,13 @@ public void record(String groupId, String artifactId, String versionId)
         getCollection().insertOne(buildDocument(metric));
     }
 
+    @Override
+    public void consolidate(VersionQueryMetric metric)
+    {
+        super.delete(getKeyFilter(metric));
```

Comment:
no business logic in the store..move this to the level up which means oyu dont need this api in the store... the fcat is called consolidate means "logic" doing more that CRUD operations